src/sagemaker/jumpstart/types.py

```diff
@@ -339,7 +339,6 @@ class JumpStartSerializablePayload(JumpStartDataHolderType):
         "content_type",
         "accept",
         "body",
-        "generated_text_response_key",
```

Comment:
question: is this backward compatible?

Reply:
yes.  we now have the `output_keys` field.